hazelcast/vector.py

```diff
@@ -68,20 +105,36 @@ class Metric(enum.IntEnum):
 
 @dataclasses.dataclass
 class IndexConfig:
+    """The IndexConfig contains configuration for a vector index.
+
+    Args:
+        name: Name of the vector index.
+        metric: The metric to be used for the index configuration.
+        dimension: The dimension of vectors to be used in the vector index.
+            All vectors that use this index configuration must have
+            the same dimension.
+        max_degree: The maximum number of connections allowed per node.
+        ef_construction: The size of the dynamic list for search.
+        use_deduplication: Enable deduplication in the index.
```

Comment:
I would rather add some explanation what deduplication means, but there is no mention of it anywhere in the docs elsewhere... (neither in the java docs)

Reply:
it is described in https://deploy-preview-1124--hardcore-allen-f5257d.netlify.app/hazelcast/5.5-snapshot/data-structures/vector-collections#configuration (not yet merged)